.travis.yml

```diff
@@ -7,7 +7,6 @@ matrix:
     - install:
       - env | grep TRAVIS > .env
       - env | grep CI >> .env
-      - docker-compose -f travis-docker-compose.yml build
```

Comment:
The build step is done automatically when `docker-compose run container` is done, if that container does not already exist. The way it's defined here it will build all containers which includes grid, but we want build only the containers that are used in the tests